cross/tmux/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = tmux
+PKG_VERS = 1.9a
+PKG_VERS_CUT = 1.9a
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://cznic.dl.sourceforge.net/project/tmux/tmux/tmux-1.9
```

Comment:
I think it'd make sense to use the direct download link, so something like `http://downloads.sourceforge.net/project/$(PKG_NAME)/$(PKG_NAME)/$(PKG_NAME)-1.9/$(PKG_DIST_NAME` might work best here.
That prevents issues with downloading when the mirror is down.